src/data/polynomial/basic.lean

```diff
@@ -173,7 +173,7 @@ def to_finsupp_iso : R[X] ≃+* add_monoid_algebra R ℕ :=
   map_mul' := by { rintros ⟨⟩ ⟨⟩, simp [mul_to_finsupp] },
   map_add' := by { rintros ⟨⟩ ⟨⟩, simp [add_to_finsupp] } }
 
-/-- Ring isomorphism between `R[X]ᵐᵒᵖ` and `R[X]ᵐᵒᵖ`. -/
+/-- Ring isomorphism between `R[X]ᵐᵒᵖ` and `Rᵐᵒᵖ[X]`. -/
```

Comment:
Maybe we should rewrite this entirely
```suggestion
/-- Ring isomorphism sending each coefficient to the opposite ring. -/
```
The currently version is just an explanation of what `≃+*` means, which is silly.